hera_pspec/plot.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pyuvdata
+from hera_pspec import conversions
+import matplotlib.pyplot as plt
+import copy
+
+"""
+def ax():
+    fig, (ax1, ax2) = plt.subplots(1, 2, sharex=True, sharey=True)
+    do_my_stuff_with_first_axis(ax=ax1)
+    do_my_stuff_with_second_axis(ax=ax2)
+"""
+
+def delay_spectrum(uvp, blpairs, spw, pol, average_blpairs=False, 
+                   average_times=False, fold=False, plot_noise=False, 
+                   delay=True, deltasq=False, little_h=True, 
+                   legend=False, ax=None):
+    """
+    Plot a 1D delay spectrum for a group of baselines.
+    
+    Parameters
+    ----------
+    uvp : UVPspec
+        UVPSpec object, containing delay spectra for a set of baseline-pairs, 
+        times, polarizations, and spectral windows.
+    
+    blpairs : list of tuples or lists of tuples
+        List of baseline-pair tuples, or groups of baseline-pair tuples.
+    
+    spw, pol : int or str
+        Which spectral window and polarization to plot.
+    
+    average_blpairs : bool, optional
+        If True, average over the baseline pairs within each group.
+        
+    average_times : bool, optional
+        If True, average spectra over the time axis. Default: False.
+    
+    fold : bool, optional
+        Whether to fold the power spectrum in |k_parallel|. Default: False.
+    
+    plot_noise : bool, optional
+        Whether to plot noise power spectrum curves or not. Default: False.
+    
+    delay : bool, optional
+        Whether to plot the power spectrum in delay units (ns) or cosmological 
+        units (h/Mpc). Default: True.
+    
+    deltasq : bool, optional
+        If True, plot dimensionless power spectra, Delta^2. This is ignored if 
+        delay=True. Default: False.
+    
+    little_h : bool, optional
```

Comment:
This should not be a kwarg, but should be inferred from the `UVPSpec` object. the power spectrum are formed with `little_h` equal to true or false, and the plotting should not alter that state. we can either 1) make a new `UVPSpec.little_h` attribute, or you can infer it from a `h` in or not in `UVPSpec.units`...